assets/js/code-title.js

```diff
@@ -0,0 +1,13 @@
+
+function addTitleToCodeBlocks() {
+  var els = document.getElementsByClassName("highlight");
+  for (var i = 0; i < els.length; i++) {
+    if (els[i].title.length) {
+      let div = document.createElement("div");
+      if (els[i].getElementsByClassName("code-title").length) continue;
```

Comment:
This line `if (els[i].getElementsByClassName("code-title").length) continue;` stops the script from rendering the code title more than once.